src/AgOledDisplay.cpp

```diff
@@ -1,5 +1,89 @@
 #include "AgOledDisplay.h"
 #include "Libraries/U8g2/src/U8g2lib.h"
+#ifdef ESP32
+#include "WiFi.h"
+#else 
+#include "ESP8266WiFi.h"
+#endif
+
+typedef struct {
+  const int width;
+  const int height;
+  const unsigned char* icon;
+} xbm_icon_t;
+
+/** WiFi ICON */
+enum {
+  WIFI_ICON_ISSUE,
+  WIFI_ICON_25,
+  WIFI_ICON_50,
+  WIFI_ICON_75,
+  WIFI_ICON_100,
```

Comment:
is 100 and MAX not the same?

Reply:
In normal develop I define _MAX  on `enum` use to define the buffer size or validate value. Like define array of icon bellow. If I don't use `[index] = {value}` I should use `wifi_icons[WIFI_ICON_MAX]`
```
const xbm_icon_t wifi_icons[] = {
  [WIFI_ICON_ISSUE] = {.width = wifi_issue_width, .height = wifi_issue_height, .icon = wifi_issue_bits},
  [WIFI_ICON_25] = {.width = wifi_25_width, .height = wifi_25_height, .icon = wifi_25_bits},
  [WIFI_ICON_50] = {.width = wifi_50_width, .height = wifi_50_height, .icon = wifi_50_bits},
  [WIFI_ICON_75] = {.width = wifi_75_width, .height = wifi_75_height, .icon = wifi_75_bits},
  [WIFI_ICON_100] = {.width = wifi_100_width, .height = wifi_100_height, .icon = wifi_100_bits},
};
```

MAX mean max item of icons

Reply:
Can we remove it?